torch/ao/quantization/observer.py

```diff
@@ -173,7 +174,7 @@ def __init__(
         eps=torch.finfo(torch.float32).eps,
     ) -> None:
         factory_kwargs = torch.nn.factory_kwargs(factory_kwargs)
-        super(_ObserverBase, self).__init__(dtype=dtype)
+        super(UniformQuantizationObserverBase, self).__init__(dtype=dtype)
```

Comment:
this can be `super().__init__(dtype=dtype)` now I think

Reply:
that's true, I just did a string replace without trying to change anything else (to minimize time + risk), let me know if you think this additional change is worth adding to this PR - I'm flexible

Reply:
sounds good, I don't have a strong preference, we can do this later